src/ui/prompts/variablePrompts.ts

```diff
@@ -132,13 +157,25 @@ export const getVariableValuePrompt = (
 ) => {
     switch (variableType) {
         case 'Boolean':
-            return variableValueBooleanPrompt(variation.key, defaultValue as boolean | undefined)
+            return variableValueBooleanPrompt(
```

Comment:
not related to your change, but we could overload the `getVariableValuePrompt` method to make sure the variable type and default value match and then remove the type casting on each of these calls

Reply:
good idea! I'll make a low prio ticket for it